data-connector-lib/src/dpk_connector/core/crawler.py

```diff
@@ -85,6 +85,15 @@ def async_crawl(
     disallow_mime_types: Collection[str] = (),
     depth_limit: int = -1,
     download_limit: int = -1,
+    concurrent_requests: int = 20,
```

Comment:
How did we come up with a default of 20 ? Also, maybe some rational for the other default values as well will be helpful. Thanks